### DISCUSSION TITLE
Feature request: Add Gravis Ultrasound ACE and Creative AWE32 Addon

### DISCUSSION THREAD

**Discussion by RAMChYLD** [86Box/86Box#3339] [Ideas]
The Gravis Ultrasound ACE is a stripped down GUS card that does not have soundblaster compatibility, it is meant to be used alongside an existing soundblaster card.

Likewise, the Creative AWE32 Addon (aka CT1920 or the Goldfinch card) card is an ISA card with only an E-Mu 8000 onboard and a set of SIMM RAM slot and nothing else, and is meant to be used alongside an existing soundblaster Pro/16 (or most likely, any other) sound card. 

I would like to propose the addition of both the abovementioned cards so it's possible to make it easier to configure a 86Box VM with either an AWE32 and a GUS ACE or a GUS with an AWE32 Upgrade card without needing to go through all sort of IRQ and DMA management.

**Comment by Torinde** [comment]
There is also [Waveblaster II](https://en.wikipedia.org/wiki/Creative_Wave_Blaster#WaveBlaster_II) (CT1910) - add-on allowing other sound cards (including not from Creative) to get EMU8k functionality via Wave Blaster header.
